src/jquery.autocomplete.js

```diff
@@ -688,7 +688,7 @@
             container.html(html);
 
             if ($.isFunction(beforeRender)) {
-                beforeRender.call(that.element, container);
+                beforeRender.call(that.element, container, that.data);
```

Comment:
Oh my god! I'm so sorry... That was my fault. I did not make the change in my cloned repo version (because it was such a small change), I mistakenly mixed the names of the variables in my application with the ones in the lib. I'll submit another request.

Ashamed.